simulation.h

```diff
@@ -108,7 +107,7 @@ class simulation
   ///Updates the inputs of the simulation with new calculated inputs
   void update_inputs(std::vector<double> new_inputs){m_input = new_inputs;}
 
-
+  void switch_env_indicator(){m_input.back() = -m_input.back();}
```

Comment:
I was thinking of something different for this. I would just use the `get_fun_name()` returned `char` and cast it to an `int` and put in the back of `input`. But this works for now, we might want to change this if we work with more than 2 functions, but this might not be the case anytime soon.

Reply:
I'll make this a future creep issue.